devel/apache-ivy/Portfile

```diff
@@ -48,6 +48,6 @@ destroot {
 }
 
 platform darwin {
-    build.env       JAVA_HOME=/Library/Java/Home
+    build.env       [exec /usr/libexec/java_home]
```

Comment:
Actually, this port should just drop the `platform darwin` block and use the [java-1.0 portgroup](https://github.com/macports/macports-ports/tree/master/_resources/port1.0/group/java-1.0.tcl), which already knows how to handle this.